src/App.css

```diff
@@ -1,4 +1,41 @@
+*,
+::before,
+::after {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
 .App {
-  display: grid;
+  text-align: center;
+  display: flex;
+  flex-direction: column;
+}
+.header {
+  position: absolute;
+  padding: 30px;
```

Comment:
I used to write px for paddings, too, but better use `em` as a unit instead of px.

Reply:
Debs your code was actually my inspiration 😅 Thanks for the tip, I'll try to use em too from now on.